test/cli/commands/make.test.ts

```diff
@@ -334,4 +301,55 @@ describe('CLI: make', () => {
     expect(upFile).to.equal(interpolate(upSQL, { table: 'settings' }));
     expect(downFile).to.equal(interpolate(downSQL, { table: 'settings' }));
   });
+
+  it('should make migration based on custom configurations with --config flag.', async () => {
```

Comment:
Can we add a few more tests in `config.test.ts` to check the contents of the config file only? 

`should load the config file that is provided`
`should load the config file only if it matches the naming convention`
`should throw an error if the config file doesn't match the naming convention`

Reply:
should load the config file (absolute path)